llama.cpp

```diff
@@ -1783,13 +1836,21 @@ struct llama_hparams {
     uint32_t n_expert_used = 0;
     uint32_t n_vocab_type = 0; // for BERT-style token types
 
+    uint32_t n_leading_dense_layer = 0;
```

Comment:
```suggestion
    uint32_t n_layer_lead = 0;
```

Reply:
Corrected, but I used n_layer_dense_lead instead of n_layer_lead (wanted to keep the info about front layer being dense)